.golangci.yml

```diff
@@ -30,6 +30,9 @@ linters-settings:
       # The io/ioutil package has been deprecated.
       # https://go.dev/doc/go1.16#ioutil
       - io/ioutil
+  forbidigo:
+    forbid:
+      - '^fmt\.Errorf(# use errors\.Errorf instead)?$'
```

Comment:
I'm not sure if it really matters, but it seems like `pkg/errors`, which is what provides `errors.Errorf` has been deprecated and archived since Nov 2021. I don't think it matters short term, but at some point it will likely become an issue and require migrating to the std library using `fmt.Errorf` and the `%w` directive for wrapping.